src/main/webapp/script.js

```diff
@@ -332,3 +347,20 @@ function createCommentElement(name, message, time) {
   commentElement.innerText = name + ' posted ' + message + ' on ' + time;
   return commentElement;
 }
+
+/**
+ * Adds ID to form submission.
+ * @return {number} Current school's ID.
+ */
+function setReviewsButton() {
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const id = urlParams.get('school-id');
+
+  const submitReviewForm = document.getElementById('submit-review');
+  submitReviewForm.setAttribute('action', `/data?school-id=${id}`);
+  const idInputElement = document.getElementById('school-id');
+  idInputElement.setAttribute('value', id);
+
+  return id;
```

Comment:
It's somewhat strange for this function to return prep the form *and* return a school ID. I think it would be good to break the ID retrieval into a separate function. I would suggest the following:

1. Add a function `getSchoolIdFromUrl`
2. Call that function from `loadComments` to get the ID
3. Make this function (`preReviewsForm` or whatever you decide to call it) accept the ID as a parameter rather than finding it itself. Pass the ID in when calling it from `loadComments`
4. Since you now would have two `getSchoolId...` methods, rename the other `getSchoolIdFromData` for clarity.